docs/kb/semgrep-ci/bitbucket-triggering-events-to-jenkins.md

```diff
@@ -0,0 +1,38 @@
+---
+tags:
+  - Bitbucket
+  - Jenkins
+  - webhooks
+description: How to trigger events from the Bitbucket server to a Jenkins instance
+---
+# Triggering events from Bitbucket to Jenkins
+
+## Motivation
+
+A common scenario for many organizations is having:
+* Bitbucket (Cloud or Server) as Source Code Management system to manage source code repositories.
+* Jenkins as CI/CD platform to run pipelines.
+So, it is crucial to have some kind of links between the two platforms.
+
+## Defining a Bitbucket webhook 
+One way to connect Bitbucket (Cloud or Server) to a Jenkins instance is defining a Bitbucket webhook.
+It will allow us to trigger builds on Bitbucket's push and pull requests events.
+
+There are two requirements to comply:
+* Installing the plugin ["Bitbucket Push and Pull Request"](https://plugins.jenkins.io/bitbucket-push-and-pull-request/) on the Jenkins server.
```

Comment:
```suggestion
* You must install the plugin [<i class="fas fa-external-link fa-xs"></i> Bitbucket Push and Pull Request](https://plugins.jenkins.io/bitbucket-push-and-pull-request/) on the Jenkins server.
```

We know have little icons that serve as signposts when users skim the docs! It looks like this on live: 

![2023-11-09_02-01-52](https://github.com/semgrep/semgrep-docs/assets/20766840/d36b0186-f7d5-4cce-a8b9-fd577718858d)